pkg/controller/master/node/node_down_controller.go

```diff
@@ -112,8 +126,12 @@ func (h *nodeDownHandler) OnNodeChanged(key string, node *corev1.Node) (*corev1.
 			return node, err
 		}
 
+		if err := h.deleteVolumeAttachmentsOfPod(&pod); err != nil {
+			return node, err
+		}
 	}
-	return node, nil
+
+	return h.resetHeartbeat(node)
```

Comment:
Should the heartbeat always need to update? In my environment, it kept renewing.

Reply:
Add a judgment that returns if detect the heartbeat has been updated yet.